app/models/user.rb

```diff
@@ -3,4 +3,10 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  has_many :microposts
+  has_many :comments
```

Comment:
**[OPTIONAL]**
It also makes sense to make microposts and comments `dependent: :destroy`, since they are related to the user.

Reply:
Done